packages/pwa-kit-create-app/assets/bootstrap/js/config/default.js.hbs

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+const sites = require('./sites.js')
+module.exports = {
+    app: {
+        // Customize how your 'site' and 'locale' are displayed in the url.
+        url: {
+            // Determine where the siteRef is located. Valid values include 'path|query_param|none'. Defaults to: 'none'
+            // site: 'none',
```

Comment:
I'm not an expert on the `config` files but the code comments have a confusing effect for me in relationship to functionality. is `// site: 'none'` being commented out because it's optional or some other reason? Should we delete entirely?

Reply:
I think that was the result of use not having a well documented API for our configuration file. The idea was to leave that comment there to show what some of the other options are. That being said this is outside of the scope of this PR as it already existed in the previous version of the generator.

Reply:
No objection

Reply:
Make sense.